src/GraphQL/Instrumentation/InstrumentFieldsMiddleware.cs

```diff
@@ -6,7 +6,14 @@ namespace GraphQL.Instrumentation
     public class InstrumentFieldsMiddleware : IFieldMiddleware
     {
         /// <inheritdoc/>
-        public async ValueTask<object?> ResolveAsync(IResolveFieldContext context, FieldMiddlewareDelegate next)
+        public ValueTask<object?> ResolveAsync(IResolveFieldContext context, FieldMiddlewareDelegate next)
+        {
+            return context.Metrics.Enabled
+                ? ResolveWhenMetricsEnabledAsync(context, next)
+                : default;
```

Comment:
```suggestion
                : next(context);
```
??

Reply:
oooooops

Reply:
I thought one thing, but wrote the other.